### ISSUE TITLE
12.35.0 appears to break the prompt

### ISSUE THREAD

**Issue by leepa** [JanDeDobbeleer/oh-my-posh#3321]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

A bug happened! I upgraded to 12.35.0. I have no custom theming  or config but the prompt is now broken. `pwshransparentm` appears on every line even with no Git remote or anything.

### Theme

Default

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 12.35.0

Segments:

ConsoleTitle(true) -  10 ms - pwsh in me
session(true)  -   0 ms -  me 
path(true)     -   1 ms -   ~
git(false)     -   6 ms -
root(false)    -   0 ms -
exit(true)     -   1 ms -   
node(false)    -   0 ms -
go(false)      -   0 ms -
python(false)  -   0 ms -
shell(true)    -   0 ms - in pwshransparentm
time(true)     -   0 ms - at 08:40:04

Run duration: 20.7781ms

Cache path: C:\Users\me\AppData\Local\oh-my-posh

Config path:

Logs:

2023/01/06 08:40:04 Getenv
C:\Users\me\AppData\Local
2023/01/06 08:40:04 Getenv: 0s, args: LOCALAPPDATA
2023/01/06 08:40:04 CachePath: 0s
2023/01/06 08:40:04 Getenv

2023/01/06 08:40:04 Getenv: 0s, args: POSH_THEME
2023/01/06 08:40:04 resolveConfigPath: 0s
2023/01/06 08:40:04 Init: 523.8µs
2023/01/06 08:40:04 Flags: 0s
2023/01/06 08:40:04 config.loadConfig: 0s
2023/01/06 08:40:04 Flags: 0s
2023/01/06 08:40:04 Getenv

2023/01/06 08:40:04 Getenv: 0s, args: OMP_CACHE_DISABLED
2023/01/06 08:40:04 WindowsRegistryKeyValue: 0s, args: HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 WindowsRegistryKeyValue
ColorizationColor(DWORD): 0xC40078D4
2023/01/06 08:40:04 Root: 0s
2023/01/06 08:40:04 Shell
process name: pwsh.exe
2023/01/06 08:40:04 Shell: 10.1956ms
2023/01/06 08:40:04 ErrorCode: 0s
2023/01/06 08:40:04 IsWsl: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 User
me
2023/01/06 08:40:04 User: 0s
2023/01/06 08:40:04 Host
JoolzDev
2023/01/06 08:40:04 Host: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 TemplateCache: 0s
2023/01/06 08:40:04 Flags: 0s
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 Getenv

2023/01/06 08:40:04 Getenv: 0s, args: SSH_CONNECTION
2023/01/06 08:40:04 Getenv

2023/01/06 08:40:04 Getenv: 0s, args: SSH_CLIENT
2023/01/06 08:40:04 TemplateCache: 0s
2023/01/06 08:40:04 TemplateCache: 0s
2023/01/06 08:40:04 Shell: 0s
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Shell: 0s
2023/01/06 08:40:04 Flags: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 PathSeparator: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 PathSeparator: 0s
2023/01/06 08:40:04 PathSeparator: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 PathSeparator: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 PathSeparator: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 PathSeparator: 0s
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 IsWsl: 0s
2023/01/06 08:40:04 StackCount: 0s
2023/01/06 08:40:04 isWriteable
not current user or in group
2023/01/06 08:40:04 isWriteable
current user is member of S-1-5-32-544
2023/01/06 08:40:04 isWriteable
WRITE_DAC
WRITE_OWNER
SYNCHRONIZE
DELETE
READ_CONTROL
2023/01/06 08:40:04 isWriteable
user has write access
2023/01/06 08:40:04 DirIsWritable: 525.6µs
2023/01/06 08:40:04 TemplateCache: 0s
2023/01/06 08:40:04 TemplateCache: 0s
2023/01/06 08:40:04 Shell: 0s
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 CommandPath
C:\Program Files\Git\cmd\git.exe
2023/01/06 08:40:04 CommandPath: 5.7976ms, args: git.exe
2023/01/06 08:40:04 HasCommand: 5.7976ms, args: git.exe
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 HasParentFilePath
CreateFile C:\.git: The system cannot find the file specified.
2023/01/06 08:40:04 HasParentFilePath: 530.9µs, args: .git
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 Root: 0s
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 ErrorCode: 0s
2023/01/06 08:40:04 TemplateCache: 0s
2023/01/06 08:40:04 TemplateCache: 0s
2023/01/06 08:40:04 Shell: 0s
2023/01/06 08:40:04 TemplateCache: 0s
2023/01/06 08:40:04 Flags: 0s
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 Shell: 0s
2023/01/06 08:40:04 Flags: 0s
2023/01/06 08:40:04 TemplateCache: 0s
2023/01/06 08:40:04 TemplateCache: 0s
2023/01/06 08:40:04 Shell: 0s
2023/01/06 08:40:04 Pwd
C:\Users\me
2023/01/06 08:40:04 Pwd: 0s
2023/01/06 08:40:04 Home
C:\Users\me
2023/01/06 08:40:04 GOOS: 0s
2023/01/06 08:40:04 TemplateCache: 0s
2023/01/06 08:40:04 TemplateCache: 0s
2023/01/06 08:40:04 Shell: 0s
2023/01/06 08:40:04 Getenv
C:\Users\me\AppData\Local
2023/01/06 08:40:04 Getenv: 0s, args: LOCALAPPDATA
2023/01/06 08:40:04 CachePath: 0s
2023/01/06 08:40:04 Flags: 0s
```

**Comment by JanDeDobbeleer** [comment]
@leepa I'll check. I did a major refactor and probably missed a use-case. That's the default initialization without config, right?

**Comment by leepa** [comment]
Yep - here's my $PROFILE

```
$env:GIT_SSH = 'C:\Windows\System32\OpenSSH\ssh.exe'
$Env:SSH_AUTH_SOCK = "\\.\pipe\ssh-pageant"

Set-PSReadlineKeyHandler -Key ctrl+d -Function ViExit

oh-my-posh init pwsh | Invoke-Expression
```

**Comment by leepa** [comment]
Can confirm - fixed in 12.35.1
